content/release-notes/1.14.1.md

```diff
@@ -0,0 +1,12 @@
+---
+date: 2020-04-01T10:00:00-04:00
+linktitle: "1.14.1"
+title: "1.14.1"
+kubernetes: "1.15, 1.16, and 1.17"
+---
+
+{{<fixes>}}
+* Clicking 'deploy' will deploy the listed version, instead of deploying the latest version.
+* Displaying an error when trying to update an airgapped install does not break the UI layout.
+* Uploading a new version on the application dashboard will not fail with a 404 error.
```

Comment:
My feedback is that the most severe issues are the airgap upload and `unable to deploy anything but latest` fixes. I'd put those at the top. I also felt that the severity wasn't too clear, so I attempted to revise wording a bit: 

* Fixed issue preventing deploys of anything other than the latest version in the Admin Console. Clicking the 'deploy' button next to an older version in the Admin Console will now correctly deploy that target version, instead of always deploying the latest version. 
* Fixed UI issue preventing all airgap bundle uploads from the dashboard. Uploads on the 'Version History' page were not affected.
* Addressed layout issues on the Admin Console dashboard that appear if there are any errors reported during airgap package upload.